client/state/application/actions.js

```diff
@@ -3,13 +3,12 @@
  */
 import { CONNECTION_LOST, CONNECTION_RESTORED } from 'state/action-types';
 import { warningNotice, successNotice, removeNotice } from 'state/notices/actions';
-import i18n from 'lib/mixins/i18n';
```

Comment:
@rralian : I had to remove it from here because tests were failing on import
Is it a known issue?